### PR TITLE
WashAnimated: rename Card to WashAnimated

### PR DESCRIPTION
**Pull request by AlbertCarreras**
#### Breaking change

Run 

```
yarn codemod renameComponent ~/path/to/your/code
  --componentName=Card
  --nextComponentName= WashAnimated
```

#### What changed?

WashAnimated: rename Card to WashAnimated

#### Why?

Differentiation from Cards in iOS and Android Platforms

### CODE REVIEW

File: packages/gestalt/src/WashAnimated.js [hunk 1]
```diff
@@ -22,21 +22,27 @@ type Props = {|
    */
   image?: Node,
   /**
-   * Optional callback fired when the user mouses over Card.
+   * Optional callback fired when the user mouses over WashAnimated.
    */
   onMouseEnter?: ({| event: SyntheticMouseEvent<HTMLDivElement> |}) => void,
   /**
-   * Optional callback fired when the user away from Card.
+   * Optional callback fired when the user away from WashAnimated.
```
Comment (dangerismycat) [inline packages/gestalt/src/WashAnimated.js:29]: looks like this has had a typo: "…when the user _mouses_ away from…"

File: packages/gestalt/src/WashAnimated.js [hunk 2]
```diff
@@ -22,21 +22,27 @@ type Props = {|
...
    */
   onMouseLeave?: ({| event: SyntheticMouseEvent<HTMLDivElement> |}) => void,
 |};
 
 /**
- * [Card](https://gestalt.pinterest.systems/web/card) is used to highlight content in grids. It visually shows that children elements belong together and can highlight the items on hover.
- * ![Card light mode](https://raw.githubusercontent.com/pinterest/gestalt/master/playwright/visual-test/Card.spec.mjs-snapshots/Card-chromium-darwin.png)
- * ![Card dark mode](https://raw.githubusercontent.com/pinterest/gestalt/master/playwright/visual-test/Card-dark.spec.mjs-snapshots/Button-dark-chromium-darwin.png)
+ * [WashAnimated](https://gestalt.pinterest.systems/web/card) is used to highlight content in grids. It visually shows that children elements belong together and can highlight the items on hover.
```
Comment (dangerismycat) [inline packages/gestalt/src/WashAnimated.js:35]: Don't forget to update this URL

File: packages/gestalt/src/index.js [hunk 3]
```diff
@@ -8,7 +8,7 @@ import Box from './Box.js';
 import Button from './Button.js';
 import ButtonGroup from './ButtonGroup.js';
 import Callout from './Callout.js';
-import Card from './Card.js';
+import WashAnimated from './WashAnimated.js';
```
Comment (dangerismycat) [inline packages/gestalt/src/index.js:11]: nit: can you alphabetize this?

File: playwright/accessibility/WashAnimated.spec.mjs [hunk 4]
```diff
@@ -2,7 +2,7 @@
 import { test } from '@playwright/test';
 import expectAccessiblePage from './expectAccessiblePage.mjs';
 
-test('Card Accessibility check', async ({ page }) => {
-  await page.goto('/web/card');
+test('WashAnimated Accessibility check', async ({ page }) => {
+  await page.goto('/web/WashAnimated');
```
Comment (dangerismycat) [inline playwright/accessibility/WashAnimated.spec.mjs:6]: this should be lowercase

**Review by dangerismycat — APPROVED** [review]

File: docs/pages/web/washanimated.js [hunk 5]
```diff
@@ -9,13 +9,13 @@ import Page from '../../docs-components/Page.js';
 import QualityChecklist from '../../docs-components/QualityChecklist.js';
 import AccessibilitySection from '../../docs-components/AccessibilitySection.js';
 import SandpackExample from '../../docs-components/SandpackExample.js';
-import defaultExample from '../../examples/card/defaultExample.js';
+import defaultExample from '../../examples/washanimated/defaultExample.js';
 
-export default function CardPage({ generatedDocGen }: {| generatedDocGen: DocGen |}): Node {
+export default function DPage({ generatedDocGen }: {| generatedDocGen: DocGen |}): Node {
```
[the remark this answers is not in the document]
Comment (AlbertCarreras) [reply docs/pages/web/washanimated.js:14]: done

File: docs/redirects.js [hunk 6]
```diff
@@ -66,6 +66,7 @@ const webComponents = [
   'upsell',
   'video',
   'zindex_classes',
+  'washanimated',
```
[the remark this answers is not in the document]
Comment (AlbertCarreras) [reply docs/redirects.js:69]: done

File: packages/gestalt/src/WashAnimated.js (hunk 1 above)
Comment (AlbertCarreras) [reply packages/gestalt/src/WashAnimated.js:29]: done

File: packages/gestalt/src/WashAnimated.js (hunk 2 above)
Comment (AlbertCarreras) [reply packages/gestalt/src/WashAnimated.js:35]: done

File: packages/gestalt/src/index.js (hunk 3 above)
Comment (AlbertCarreras) [reply packages/gestalt/src/index.js:11]: done

File: playwright/accessibility/WashAnimated.spec.mjs (hunk 4 above)
Comment (AlbertCarreras) [reply playwright/accessibility/WashAnimated.spec.mjs]: DONE

**Pull request merged**